src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const splitedString = sourceString.split(';').map(words =>
```

Comment:
Split is irregular verb in English :)

```suggestion
  const splitString = sourceString.split(';').map(words =>
```